src/v0/destinations/google_adwords_enhanced_conversions/networkHandler.js

```diff
@@ -25,6 +28,7 @@ const tags = require('../../util/tags');
  */
 const getAuthErrCategory = (code, response) => {
```

Comment:
Can we make `getAuthErrCategory` a global utility? The logic determining the error category is "almost" duplicated in many destinations.

Reply:
We can put this as some sort of google utility(that encompasses a lot of google related destinations) not a global one

Reply:
Moved the functions to utility. Can you once check and confirm ?